src/Uno.UWP/System.IO/WindowsRuntimeStorageExtensions.cs

```diff
@@ -13,7 +13,7 @@ public static async Task<Stream> OpenStreamForReadAsync(this IStorageFile window
 		{
 			if (windowsRuntimeFile is StorageFile file)
 			{
-				return await file.OpenStream(CancellationToken.None, FileAccessMode.Read, StorageOpenOptions.None);
+				return await file.OpenStream(CancellationToken.None, FileAccessMode.Read, StorageOpenOptions.AllowReadersAndWriters);
```

Comment:
LGTM ... but did you validate that behavior on windows? (Same for the `OpenForWrite`)